app.js

```diff
@@ -34,12 +34,12 @@ require('./passportConfig')(passport);
 const loginController = require('./controllers/loginController');
 const registerController = require('./controllers/registerController');
 const getUserInfoController = require("./controllers/getUserInfoController");
-const getLoggedInController = require("./controllers/getLoggedInController");
+const getLoggedInUserController = require("./controllers/getLoggedInController");
 
 
 app.post("/user", registerController);
 app.post("/user/login", loginController);
-app.get("/user/loggedIn", getLoggedInController);
+app.get("/user/loggedIn", getLoggedInUserController);
 app.get("/user/:user", getUserInfoController);
 
 
```

Comment:
Good change but the filename isn't changed - could you change that to match?

Reply:
No